faithcomesbyhearing/dbp
# FCBH-488 Daily synchronization of user data from v2 to v4 servers

# Description
- Updated kernel functions in order to allow the synchronization of the v2 data
   - Users 
   - Profiles
   - Bookmarks 
   - Notes
   - Highlights
- Updated `.ENV` example file

## Issue Link
Original Story: [FCBH-488](https://fullstacklabs.atlassian.net/browse/FCBH-488) 
Review Story: [FCBH-811](https://fullstacklabs.atlassian.net/browse/FCBH-811) 

## How Do I QA This
- Add `V2 Database connections` credentials to the `.env` on your local environment
- Do initial sync running 
    1. `php artisan syncV2:users 2000-01-01`
    2. `php artisan syncV2:profiles 2000-01-01`
    3. `php artisan syncV2:bookmarks 2000-01-01`
    4. `php artisan syncV2:highlights 2000-01-01`
    5. `php artisan syncV2:notes 2000-01-01`
- Then you can run the command without a startDate (The cron job will do this also), this will only sync the entries of the current day
    1. `php artisan syncV2:users`
    2. `php artisan syncV2:profiles`
    3. `php artisan syncV2:bookmarks`
    4. `php artisan syncV2:highlights`
    5. `php artisan syncV2:notes`
- Add this cron job to your environment
  `* * * * * cd /path-to-your-project && php artisan schedule:run >> /dev/null 2>&1`
- Verify that now the v2 data synchronization is working

## Checklist
